Migration script/openIMIS migration latest.sql

```diff
@@ -7297,12 +7297,12 @@ BEGIN
 END
 GO
 
-IF OBJECT_ID('uspAPIGetCoverage', 'P') IS NOT NULL
-    DROP PROCEDURE uspAPIGetCoverage
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
 GO
-CREATE PROCEDURE [dbo].[uspAPIGetCoverage]
+CREATE OR ALTER PROCEDURE [dbo].[uspAPIGetCoverage]
```

Comment:
Please drop procedure before as this method is not supported in MSSQL 2012.

Reply:
Fixed